benchexec/tools/cbmc.py

```diff
@@ -130,13 +137,13 @@ def determine_result(self, returncode, returnsignal, output, isTimeout):
                 elif "UNKNOWN" in output:
                     status = result.RESULT_UNKNOWN
 
-        elif returncode == 64 and "Usage error!\n" in output:
+        elif run.exit_code.value == 64 and "Usage error!\n" in output:
```

Comment:
Lines with line separators won't ever be in `run.output`, so `\n` needs to be removed from these and other strings.

Reply:
fixed in a6dbfcbc3fefc99d6a7a99e9a4040b7dee88e614